src/odemis/acq/stream_settings.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+"""
+Created on 24 Jan 2024
+
+@author: Karishma Kumar
+
+Copyright © 2024 Karishma Kumar, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License version 2 as published by the Free Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+import json
+import logging
+import os
+from typing import List, Dict, Any
+
+from odemis import model, util
+from odemis.acq.stream import FluoStream
+from odemis.util.img import md_format_to_tint, tint_to_md_format
+
+SETTINGS_ORDER = ["name", "excitation", "power", "emission", "tint", "detExposureTime", "auto_bc_outliers"]
+NON_SETTINGS_VA = ['acquisitionType', 'auto_bc', 'background', 'histogram', 'image', 'intensityRange', 'is_active',
+                   'roi', 'should_update', 'single_frame_acquisition', 'status']
+
+
+def get_config_index(config_data: list, name: str) -> int:
+    """
+    Get the index of the config entry with the given name
+    :param config_data: list of config entries
+    :param name: name of config entry
+    :return: index of config entry with given name
+    """
+    try:
+        index = list(map(lambda x: x["name"], config_data)).index(name)
```

Comment:
Often a list comprehension is easier to read than `map`. Here, I think this is easier:
```suggestion
        index = [x["name"] for x in config_data].index(name)
```